rpcs3/Emu/Cell/lv2/sys_game.cpp

```diff
@@ -35,8 +105,3 @@ error_code _sys_game_board_storage_read(vm::ptr<u8> buffer1, vm::ptr<u8> buffer2
 
 	return CELL_OK;
 }
-
-u64 _sys_game_get_system_sw_version()
```

Comment:
Why is this moved?

Reply:
I just re-arranged the order according to the order of sys_game syscalls in lv2.cpp.